scripts/generate-test-services.ts

```diff
@@ -170,7 +170,10 @@ async function generateAll(): Promise<void> {
   }
 
   if (arg === 'openapi' || arg === 'rest' || arg === 'all') {
-    await generateOpenApi(generatorConfigOpenApi);
+    await generateOpenApi({
+      ...generatorConfigOpenApi,
+      generateJs: true
+    });
```

Comment:
Question: why we want the JS here?

Reply:
See [here](https://github.com/SAP/cloud-sdk-js/pull/1069#discussion_r592326926).